defrag/tests/unittests/test_news_reddit_twitter.py

```diff
@@ -0,0 +1,34 @@
+# Defrag - centralized API for the openSUSE Infrastructure
```

Comment:
I do not really see why you moved these files, if you really want to keep them here, please create a `__init__.py` file in `defrag/tests/`, otherwise all tests will fail like this:

```
================================================================================ test session starts ================================================================================
platform linux -- Python 3.9.5, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
rootdir: /data/jens/Development/defrag/defrag/tests
plugins: asyncio-0.15.1
collected 0 items / 2 errors                                                                                                                                                        

====================================================================================== ERRORS =======================================================================================
_____________________________________________________________________ ERROR collecting unittests/test_cache.py ______________________________________________________________________
ImportError while importing test module '/data/jens/Development/defrag/defrag/tests/unittests/test_cache.py'.
Hint: make sure your test modules/packages have valid Python names.                                                                                                                  
Traceback:                                                                                                                                                                           
/usr/lib64/python3.9/importlib/__init__.py:127: in import_module                                                                                                                     
    return _bootstrap._gcd_import(name[level:], package, level)                                                                                                                      
unittests/test_cache.py:17: in <module>                                                                                                                                              
    from defrag.modules.helpers.caching import cache                                                                                                                                 
E   ModuleNotFoundError: No module named 'defrag'                                                                                                                                    
______________________________________________________________ ERROR collecting unittests/test_news_reddit_twitter.py _______________________________________________________________
ImportError while importing test module '/data/jens/Development/defrag/defrag/tests/unittests/test_news_reddit_twitter.py'.
Hint: make sure your test modules/packages have valid Python names.                                                                                                                  
Traceback:                                                                                                                                                                           
/usr/lib64/python3.9/importlib/__init__.py:127: in import_module                                                                                                                     
    return _bootstrap._gcd_import(name[level:], package, level)                                                                                                                      
unittests/test_news_reddit_twitter.py:17: in <module>                                                                                                                                
    from defrag.modules.news_reddit_twitter import RedditStore, TwitterStore                                                                                                         
E   ModuleNotFoundError: No module named 'defrag'                                                                                                                                    
============================================================================== short test summary info ==============================================================================
ERROR unittests/test_cache.py
ERROR unittests/test_news_reddit_twitter.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================================= 2 errors in 0.09s =================================================================================
```

Reply:
I moved them because unittests is just one kind of tests we want to perform. There are integration tests too. Separating the two will make refactoring easier in the future.